aws/networking/main.tf

```diff
@@ -0,0 +1,79 @@
+data "aws_availability_zones" "available" {
+  state = "available"
+}
+
+resource "aws_vpc" "main" {
+  cidr_block           = "10.0.0.0/16"
```

Comment:
This should probably be an input with a default value.

Reply:
Agreed.